core/src/main/java/hudson/node_monitors/DiskSpaceMonitor.java

```diff
@@ -61,7 +63,17 @@ public String getColumnCaption() {
         return Jenkins.get().hasPermission(Jenkins.ADMINISTER) ? super.getColumnCaption() : null;
     }
 
-    public static final DiskSpaceMonitorDescriptor DESCRIPTOR = new DiskSpaceMonitorDescriptor() {
+    @SuppressFBWarnings(value = "MS_PKGPROTECT", justification = "for backward compatibility")
+    public static /*almost final*/ DiskSpaceMonitorDescriptor DESCRIPTOR;
+
+    @Extension @Symbol("diskSpace")
+    public static class DescriptorImpl extends DiskSpaceMonitorDescriptor {
```

Comment:
Unsure whether we want `@since` for this. Technically, yes?

Reply:
I don't see really the need for this here.